node-sdk-sample/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "sdk-sample",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "commonjs",
+  "directories": {
+    "test": "tests"
+  },
```

Comment:
Curious about this as I haven't seen a `directories` section in a package.json file before 🤔